lib/multivm/src/lib.rs

```diff
@@ -22,6 +25,33 @@ pub enum ExecutionVersion {
     V3 = 3,
 }
 
+// Verification Key Hash and Execution Version have a 1-to-1 mapping starting from V3 onwards.
+// We could backport for V1 and V2 if needed in the future, but it's very unlikely.
```

Comment:
I hardly find any value in backporting, as we'll be using the "vk versioned" system from V3 onwards. That said, happy to hear more thoughts.